src/main/kotlin/uk/gov/justice/digital/hmpps/prison/visitbooker/registry/model/repository/PermittedPrisonerRepository.kt

```diff
@@ -1,12 +1,25 @@
 package uk.gov.justice.digital.hmpps.prison.visitbooker.registry.model.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
 import uk.gov.justice.digital.hmpps.prison.visitbooker.registry.model.entity.PermittedPrisoner
 
 @Repository
 interface PermittedPrisonerRepository : JpaRepository<PermittedPrisoner, Long> {
   fun findByBookerId(bookerId: Long): List<PermittedPrisoner>
   fun findByBookerIdAndActive(bookerId: Long, active: Boolean): List<PermittedPrisoner>
   fun findByBookerIdAndPrisonerId(bookerId: Long, prisonerId: String): PermittedPrisoner?
+
+  @Transactional
+  @Modifying
```

Comment:
Do we need a Modifying annotation ? Is it because the service layer sets the active to true?

Why would we not do an UPDATE query ?